### PR TITLE
Hotfix for unexpected bugs from #2

### PR DESCRIPTION
**Pull request by chuanhao01**
# Hotfix for unexpected bugs from #2

As mentioned in #2, github pages introduced some unexpected bugs.
Hopefully, the review below should fix all of them

### CODE REVIEW

File: 07-advanced-git/07-01-git-log/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Now that you know the basics of Git and have started working on the repository w
 
 This is where this chapter, and the subsequent (`Git Diffs`), come in. These commands give you a way to visualize the commits and changes made in your Git repository.  
 
-**TL|DR:** The command `git log` and its corresponding optional flags lets you look at the history of commits and how they link up in your repository.  
+**TLDR:** The command `git log` and its corresponding optional flags lets you look at the history of commits and how they link up in your repository.
```
Comment (chuanhao01) [inline 07-advanced-git/07-01-git-log/README.md:7]: Github Pages fix, unexpected rendering of table in MD

File: 07-advanced-git/README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # Table of contents
 
-1. [`Git Log`](./07-01-git-log/README.md)
+1. [Git Log](./07-01-git-log/README.md)
```
Comment (chuanhao01) [inline 07-advanced-git/README.md:3]: Changing format to be consistent with root README.md  

Can change this if we want to make the root follow this format instead

Comment (woojiahao) [reply 07-advanced-git/README.md:3]: Either is fine with me, as long as both are consistent :+1:

**Review by woojiahao — APPROVED** [review]
Looks good, let me know if you do spot any other issues otherwise I will merge these changes in the evening or tomorrow morning :+1:

**Comment by chuanhao01** [comment]
Don't think I saw any others, so feel free to merge. 👍

**Comment by woojiahao** [comment]
Awesome stuff, good work @chuanhao01 ! :clap:

**Pull request merged**